addons/hr_timesheet/views/project_task_views.xml

```diff
@@ -107,30 +107,28 @@
                     <group invisible="not analytic_account_active">
                         <group class="oe_subtotal_footer" name="project_hours">
                             <span class="o_td_label float-start">
-                                <label class="fw-bold" for="effective_hours" string="Hours Spent" invisible="encode_uom_in_days"/>
-                                <label class="fw-bold" for="effective_hours" string="Days Spent" invisible="not encode_uom_in_days"/>
+                                <label class="fw-bold" for="effective_hours" string="Time Spent"/>
                             </span>
                             <field name="effective_hours" widget="timesheet_uom" nolabel="1"/>
                             <button name="action_view_subtask_timesheet" type="object" class="ps-0 border-0 oe_inline oe_link mb-2 o_td_label float-start" invisible="subtask_effective_hours == 0.0">
-                                <span class="text-nowrap" invisible="encode_uom_in_days">Hours Spent on Sub-tasks:</span>
-                                <span class="text-nowrap" invisible="not encode_uom_in_days">Days Spent on Sub-tasks:</span>
+                                <span class="text-nowrap">Time Spent on Sub-tasks:</span>
                             </button>
                             <field name="subtask_effective_hours" class="mt-2" widget="timesheet_uom"
                                    invisible="subtask_effective_hours == 0.0" nolabel="1"/>
                             <span invisible="subtask_effective_hours == 0.0" class="o_td_label float-start my-1">
-                                <label class="fw-bold" for="total_hours_spent" string="Total Hours"
+                                <label class="fw-bold" for="total_hours_spent" string="Total Time Spent"
                                        invisible="subtask_effective_hours == 0.0 or encode_uom_in_days"/>
                                 <label class="fw-bold" for="total_hours_spent" string="Total Days"
                                        invisible="subtask_effective_hours == 0.0 or not encode_uom_in_days"/>
```

Comment:
Those two labels should be unified. Also, again, I don't think `string` is needed anymore